### PR TITLE
i18n - pulling from tx

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: locale/action_names.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Autogenerated!
 _("Action with sub plans")
+_("Check for long running tasks")
+_("Deliver notifications about long running tasks")
```
[the remark this answers is not in the document]
Comment (adamruzicka) [reply locale/action_names.rb:4]: That's from tasks, https://github.com/theforeman/foreman-tasks/blob/master/app/lib/actions/check_long_running_tasks.rb#L42 and https://github.com/theforeman/foreman-tasks/blob/master/app/lib/actions/deliver_long_running_tasks_notification.rb#L19

[the remark this answers is not in the document]
Comment (stejskalleos) [reply locale/action_names.rb:4]: This is how I updated the translations:

* `mkdir -p locale/{en,fr,ja,zh_CN,zh_TW,pt_BR}`
* `make -C locale tx-update` (In plugin dir)
From foreman:
Edit gem file (only one plugin enabled)
* `bundle;be rails db:drop;be rails db:create;be rails db:migrate`
* `P="foreman_leapp";be rake "plugin:gettext[$P]" && be rake "plugin:po_to_json[$P]"`
From plugin dir:
* `make -C locale mo-files`

[the remark this answers is not in the document]
Comment (stejskalleos) [reply locale/action_names.rb:4]: No plugin except foreman leapp is enabled

[the remark this answers is not in the document]
Comment (adamruzicka) [reply locale/action_names.rb:4]: > No plugin except foreman leapp is enabled

leapp depends on rex and ansible, both depend on tasks.

[the remark this answers is not in the document]
Comment (ekohl) [reply locale/action_names.rb:4]: Ah ok, then it makes sense. This does highlight that it becomes more important to somehow be able to filter those tasks out.

[the remark this answers is not in the document]
Comment (stejskalleos) [reply locale/action_names.rb:4]: SO can we merge?

**Review by ekohl — CHANGES_REQUESTED** [review]
It looks like Japanese is missing, which does have translations in Transifex. Perhaps that was fixed when I resynced with the proper encoding. Mind pulling again to make sure?

**Comment by stejskalleos** [comment]
Updated the strings but it looks like Japanese translation is still empty

**Pull request closed without merging**